tests/integration/cgroups.bats

```diff
@@ -190,6 +190,22 @@ function setup() {
 	[[ "$output" == *'invalid configuration'* ]]
 }
 
+@test "runc run (blkio weight)" {
+	requires root cgroups_v2
```

Comment:
Shouldn't require root?

Reply:
@AkihiroSuda once https://github.com/opencontainers/runc/pull/2818 is merged we will be able to enable it (rm root, add `[[ "$ROOTLESS" -ne 0 ]] && requires rootless_cgroup`)

Reply:
...because currently `runc exec test_cgroups_unified sh -c 'cat /sys/fs/cgroup/io.bfq.weight'` is not working